ctx_test.go

```
@@ -5931,6 +5931,68 @@
 	require.Nil(t, resp)
 }
 
+// go test -run Test_Ctx_End
+func Test_Ctx_End(t *testing.T) {
+	app := New()
+
+	app.Get("/", func(c Ctx) error {
+		c.SendString("Hello, World!")
```

golangci-lint (1.62.2)

Error return value of `c.SendString` is not checked

(errcheck)